src/components/ActionsPanel/components/ActionItem.tsx

```diff
@@ -20,7 +20,7 @@ const greenLightup = keyframes`
   }
 `;
 
-const Action = styled.div<{ state?: string }>`
+export const Action = styled.div<{ state?: string }>`
```

Comment:
Why the export?

Reply:
It was used for something I removed, fixing it